x-pack/plugins/security_solution/public/management/components/effected_policy_select/effected_policy_select.tsx

```diff
@@ -215,28 +217,32 @@ export const EffectedPolicySelect = memo<EffectedPolicySelectProps>(
                 onChange={handleGlobalButtonChange}
                 color="primary"
                 isFullWidth
+                data-test-subj={getTestId('byPolicyGlobalButtonGroup')}
               />
             </EuiFormRow>
           </EuiFlexItem>
         </EuiFlexGroup>
         <EuiSpacer />
-        {!isGlobal && (
-          <EuiFormRow fullWidth>
-            <StyledEuiSelectable>
-              <EuiSelectable<OptionPolicyData>
-                {...otherSelectableProps}
-                options={selectableOptions}
-                listProps={listProps || DEFAULT_LIST_PROPS}
-                onChange={handleOnPolicySelectChange}
-                searchProps={SEARCH_PROPS}
-                searchable={true}
-                data-test-subj={getTestId('policiesSelectable')}
-              >
-                {listBuilderCallback}
-              </EuiSelectable>
-            </StyledEuiSelectable>
-          </EuiFormRow>
-        )}
+        {!isGlobal &&
+          (isLoading ? (
```

Comment:
this could be in the same conditional clause no? `!isGlobal && isLoading ? (<Loader>) : (<EuiFormRow>)`

Reply:
Not at all, the `else` clause has to be displayed only if it's not global, otherwise it has to be undefined.

Reply:
I see! 👍